projects/ngx-konva/src/lib/directives/ko-transform.directive.ts

```diff
@@ -45,8 +58,19 @@ export class KoTransformDirective implements OnInit, OnDestroy {
     this.node.off('transformstart', this.onTransformStarterListener);
     this.node.off('transformend', this.onTransformEndListener);
     this.node.off('transform', this.onTranformListener);
+    this.removeTransformer();
+  }
+
+  addTransformer() {
```

Comment:
Set private (remove transformer too)
Add a fail safe in the case the Transformer is already set (to avoid memory leaks)
EG remove the current first or throw an error if it can initialized only once

Reply:
Done in second commit.